docs/UserGuide.md

```diff
@@ -515,10 +516,24 @@ Format: `alias TARGET ALIAS`
 
 Example:
 
-* `alias switch st` ---  This will add `st` as a valid shortcut for command
-  `switch`.
-* `alias addj j` --- This will add `j` as a valid shortcut for command `addj`
+1. Assuming you want to add `st` as an alias for `switch`, type `alias switch st` 
+into the command box and press enter.
 
+   ![alias example](images/UGExamples/aliasExample1.png)
+   
+2. If you enter a valid alias, you can see the message from IntelliJournal which tells you the operation is successful.
```

Comment:
Same thing here, there should be no ifs here, since this example will definitely be successful.
So maybe change it to:
**This adds `st` as a valid shortcut for the command `switch`**